libs/store/src/lib/loggers/player.ts

```diff
@@ -0,0 +1,36 @@
+import { AppErrorCode, AppMessageLogType } from '@razor/models';
+import { Dispatch } from '../store';
+
+export const playerNotFound = async (
+  dispatch: Dispatch,
+  pid: string,
+  additionalMessage?: string,
+) => {
+  dispatch.game.sendLogMessage({
+    message: `Player with id ${pid} does not exist`,
+    code: AppErrorCode.PlayerNotExists,
+    related: additionalMessage ? additionalMessage : '',
+    type: AppMessageLogType.Error,
+  });
+  return;
+};
+
+export const invalidPlayerName = async (dispatch: Dispatch) => {
+  dispatch.game.sendLogMessage({
```

Comment:
Do we need to `await` on the result?
If not, why do we have specified an `async` function?

Reply:
Fixed in PR #14 under commit 563c7da7291336f0b666f5c471e3d5046054c636